src/Services.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Interface BackgroundTask.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AmpProject\AmpWP\BackgroundTask\MonitorCssTransientCaching;
+
+/**
+ * Centralized service handler to simplify management of registration, activation and deactivation hooks.
+ *
+ * @package AmpProject\AmpWP
+ */
+final class Services {
+
+	/**
+	 * List of known services.
+	 *
+	 * @var string[]
+	 */
+	const ALL = [
+		MonitorCssTransientCaching::class,
+	];
```

Comment:
Eventually would this be specified externally? It seems a bit weird to put them in here.